netsim-core/src/policy.rs

```diff
@@ -92,7 +92,7 @@ impl Latency {
         Self(duration)
     }
 
-    pub(crate) fn to_duration(self) -> Duration {
+    pub fn to_duration(self) -> Duration {
```

Comment:
The `geo::latency_between_locations`  function returns this opaque `Latency` struct, and I wanted to add extra latency between some nodes. So I publicly exposed this `to_duration` method so consumers can use the result in computations.